src/reputation/distributions.py

```diff
@@ -16,6 +23,90 @@ def amount(self):
     def gives_rep(self):
         return self._give_rep
 
+RSC_YEARLY_GIVEAWAY = 50000000
+MINUTES_IN_YEAR = 525960
```

Comment:
Is this a mistake? there are 525,600 minutes in one year

Reply:
Yes that is a mistake

Reply:
Well, google tells me there are 525960 minutes in a year.

Reply:
<img width="756" alt="Screen Shot 2022-04-07 at 9 43 53 AM" src="https://user-images.githubusercontent.com/2453737/162254781-94b56e69-93ae-4fca-96f9-d09ee81a055f.png">

Reply:
lol interesting. It's pretty minor. I just multiplied 60*24*365. Maybe it is the average between regular year and leap year or something like that.